### PR TITLE
Sort Array of 0's 1's 2's in C++

### LINKED ISSUE #1161: Adding Sort an array of 0s, 1s and 2s in C++

### PR DESCRIPTION
**Pull request by HariniJeyaraman**
Fixes #1161

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
-  Added implementation of Sort in array containing only 0's 1's and 2's in C++
- 

#### Languages Used:
-  C++
- 

#### Files Added:
-  Sort_Array_zero_one_two.cpp
- 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by HariniJeyaraman** [comment]
@jainaman224 @SakshiShreya please review the code . Implemented in C++ to segregate 0's 1's and 2's in the array.

**Comment by jainaman224** [comment]
Complete PR template. Issue number is missing.

**Comment by HariniJeyaraman** [comment]
@jainaman224 @SakshiShreya  updated the PR template. Filled the issue number, file name.

File: Sort_Array_zero_one_two/segregate_zero_one_two.cpp [hunk 1]
```diff
@@ -0,0 +1,51 @@
+/*Given an array A[] consisting 0s, 1s and 2s. The task is to write a function that sorts the given array.
```
Comment (jainaman224) [inline Sort_Array_zero_one_two/segregate_zero_one_two.cpp]: Fix file and folder name.

File: Sort_Array_zero_one_two/segregate_zero_one_two.cpp [hunk 2]
```diff
@@ -0,0 +1,51 @@
+/*Given an array A[] consisting 0s, 1s and 2s. The task is to write a function that sorts the given array. 
+The function puts all 0s first, then all 1s and all 2s in last.*/
+
+#include<stdlib.h>
+#include<iostream>
+using namespace std;
+
+void sort_zero_one_two(int a[],int n)
+{
+    int zero=0; //counter to store number of 0's
```
Comment (jainaman224) [inline Sort_Array_zero_one_two/segregate_zero_one_two.cpp]: Add spaces around operators like `+`, `=`.

File: Sort_Array_zero_one_two/segregate_zero_one_two.cpp [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        cout<<a[i]<<" ";
+    cout<<"\n";
+    return 0;
+}
+
+/* Sample Input - Output
+Enter the number of elements of array: 7                                                                                                                                                                                                                  
+Enter the array elements: 1 0 2 1 1 0 2                                                                                                                                                                                                             
+Original array : 1 0 2 1 1 0 2 
+Sorted array   : 0 0 1 1 1 2 2  */
```
Comment (jainaman224) [inline Sort_Array_zero_one_two/segregate_zero_one_two.cpp]: Add single new line at the end of file.

File: Sort_Array_zero_one_two/segregate_zero_one_two.cpp [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    for(int i=0;i<n;i++)
+        cin>>a[i];
+    cout<<"Original array : ";
+    for(int i=0;i<n;i++)
+        cout<<a[i]<<" ";
+    sort_zero_one_two(a,n); //calling the sort function
+    cout<<"\nSorted array   : ";
+    for(int i=0;i<n;i++)
+        cout<<a[i]<<" ";
+    cout<<"\n";
```
Comment (jainaman224) [inline Sort_Array_zero_one_two/segregate_zero_one_two.cpp]: Use endl.

**Comment by HariniJeyaraman** [comment]
What should be the file and folder name

**Comment by HariniJeyaraman** [comment]
Fixed the file and folder name, added spaces around operators, added blank link at end of file, used endl instead of \n. @jainaman224  please review again.

**Pull request merged**